src/transformers/models/git/modeling_git.py

```diff
@@ -1438,6 +1438,28 @@ def forward(
         >>> np.random.seed(45)
 
 
+        >>> def read_video_pyav(container, indices):
+        ...     '''
+        ...     Decode the video with PyAV decoder.
+        ...     Args:
+        ...         container (container): PyAV container.
+        ...         start_index (int): the starting frame.
```

Comment:
`indices` in fn signature here not same as docstring (`start_index` / `end_index`).

Reply:
Applies to all other copies of this snippet below too (won't comment on each but wanted to make sure I noted this).